AboutDoc/Documentation_About.rst

```diff
@@ -0,0 +1,250 @@
+|  Github on Pull requests https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork
+|  Github text howto https://opensource.com/life/16/3/submit-github-pull-request
+|  Github video howto https://www.youtube.com/watch?v=rgbCcBNZcdQ
+
+It is possible to edit files directly in the Github web interface.
+To edit a file via the web interface,
+you can just click the pencil icon in its upper-right corner.
+
+When you started editing,
+Github would see that you do not have permissions to make changes directly to files here.
+So it would set you up with a copy ("fork") under your account,
+where you can make changes before submitting them as a Pull Request.
+
+Editing on the web is perfectly workable.
+The only downside being that there is no real way to preview your changes.
+With a local clone you can use a previewing editor or
+(if you have the necessary Sphinx tools installed)
+generate updated HTML docs and view them in a web browser.
+
+Sphinx
+------
+`Spinx <https://en.wikipedia.org/wiki/Sphinx_(documentation_generator)>`_ was created to simply generate documentation from Python sourcecode.
+It is written in Python, and also used in other environments. 
+It is licensed under the BSD license.
+It can also export files for translation.
+
+Generating a local copy of the manual requires only the Python-based Sphinx documentation system and the Sphinx RTD theme.  
+They can be installed  using most package managers, or via 
+
+.. code-block:: console
+
+  pip3 install sphinx sphinx-rtd-theme
+
+Anyone who would like to contribute and needs help with installing and using Sphinx can ask for support in the issues tracker.
+
+Tutorial video:	https://www.youtube.com/watch?v=ouHVkMo3gwE
+
+ReST Basic Syntax
+-----------------
+`List of basic syntax </Documentation_RestSyntax.rst>`_  in Openshot documentation.  
+
+- Some explanation here:  https://hyperpolyglot.org/lightweight-markup
+- or here: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html
+- Video tutorials here:  https://www.youtube.com/results?search_query=restructuredtext+tutorial
+
+Software 
+
+- Notepad++ ReST template:	https://github.com/steenhulthin/reStructuredText_NPP
+- Linux: 
+
+File naming and directory structure
+-----------------------------------
+
+| Files are stored in \docs
+| File names may not contain spaces. Spaces in filenames cause problems with Sphinx. 
+| For multi-word filenames, ReST documentation filenames are separated by an underscore ( _ ).
+| Image filenames are separated by a dash ( - ). 
+| Images are named after the file they belong to, and sorted in the doc\\images subdirectory. 
+
+.. NOTE: Or maybe not, see Images
+
+Translations go into their own directory. 
+
+File structure
+--------------
+
+Every file starts with 5 hidden segments (See `template <Template.rst>`_ )
+
+- Copyright notice
+- Openshot description
+- Openshot disclaimer
+- License notice
```

Comment:
These are really all just one four-paragraph block, our standard header. (It serves as all of those things, but they shouldn't be considered separate things.)